frontend/next.js/BUILD.bazel

```diff
@@ -80,3 +81,66 @@ build_test(
         # ":next_export",
     ],
 )
+
+platform(
+    name = "linux_amd64",
+    constraint_values = [
+        "@platforms//os:linux",
+        "@platforms//cpu:x86_64",
+    ],
+)
+
+platform(
+    name = "linux_arm64",
+    constraint_values = [
+        "@platforms//os:linux",
+        "@platforms//cpu:arm64",
+    ],
+)
+
+next_bin.next_binary(
+    name = "next_js_start",
+    fixed_args = ["start"],
+    chdir = package_name(),
+    data = [
+        "//next.js/pages",
+        "//next.js/public",
+        "//next.js/styles",
+        ":next",
+        "next.config.js",
+        "package.json",
+        ":node_modules/@bazel-example/one",
+        ":node_modules/is-even",
+        ":node_modules/next",
+        ":node_modules/react",
+        ":node_modules/react-dom",
+        ":node_modules/typescript",
+    ],
+)
+
+js_image_layer(
+    name = "layers",
+    binary = ":next_js_start",
+    platform = select({
```

Comment:
For transitioning the target platform to linux for for the container runtime so the image can be built locally on a mac